src/main/java/lotto/type/RankType.java

```diff
@@ -0,0 +1,41 @@
+package lotto.type;
+
+public enum RankType {
+    NO_RANK(0, 0),
+    RANK1(2000000000, 6),
```

Comment:
```suggestion
    RANK1(2_000_000_000, 6),
```
돈과 같이 1000단위로 구분은 다음과 같이 underbar(_)를 활용해 구현하면 코드 가독성이 좋아집니다.